x-pack/plugins/apm/public/components/app/error_group_details/detail_view/index.tsx

```diff
@@ -157,9 +157,9 @@ export function DetailView({ errorGroup, urlParams }: Props) {
             <EuiTab
               onClick={() => {
                 history.replace({
-                  ...location,
+                  ...history.location,
```

Comment:
It's a bummer that we can't catch this bug statically. This issue seems to be exactly that: https://github.com/microsoft/TypeScript/issues/14306

This could be a solution:
https://github.com/microsoft/TypeScript/issues/14306

But it looks like we are already setting `no-restricted-globals`:
https://github.com/elastic/kibana/blob/b0b0584b55b7b48057d806436ba7fe69da7f1567/packages/elastic-eslint-config-kibana/javascript.js#L67

Reply:
@sqren the rules is only applied for js files https://github.com/elastic/kibana/blob/master/packages/elastic-eslint-config-kibana/javascript.js#L10. I'm going to open a PR that adds this rule in the TS files https://github.com/elastic/kibana/blob/master/packages/elastic-eslint-config-kibana/typescript.js